en/peripherals/camera.md

```diff
@@ -33,17 +33,20 @@ Mode | Description
 
 ## Trigger Hardware Configuration {#hardware_setup}
 
-You can choose which pins to use for triggering using the [TRIG_PINS](../advanced_config/parameter_reference.md#TRIG_PINS) parameter.
+On Pixhawk FMUv5x boards use the board-specific camera capture pin (PI0) for triggering
+(this is automatically enabled when `TRIG_INTERFACE` is 0/GPIO or 1:Seagull).
```

Comment:
@bkueng is this line correct? Or is this also enabled if PWM interface is selected, so the line should be "(this is automatically enabled when `TRIG_INTERFACE` not set to MAVLink"?